packages/devops/scripts/deployment/restoreE2eDataFromS3.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
This will be run on startup, to restore e2e test data for any deployment with a branch ending in `e2e`